pulse_actions/worker.py

```diff
@@ -49,10 +50,10 @@ def run_pulse(exchange, topic, event_handler, topic_base, dry_run=True):
     def handler_with_dry_run(data, message):
         return event_handler(data, message, dry_run)
 
-    pulse = PulseConsumer(exchange,
+    pulse = PulseConsumer(exchanges,
                           callback=handler_with_dry_run,
                           **pulse_args)
-    LOG.info('Listening on %s, with topic %s', exchange, topic)
+    LOG.info('Listening on %s, with topic %s', exchanges, topics)
 
     while True:
         pulse.listen()
```

Comment:
After merging this, I would like to deploy to Heroku directly from master without changing the code. That means we need to add some error-safeness here. Can you put pulse.listen() inside a try block that just logs the exception when one happens? That way we can collect all the errors that show up without having to restart the dynos all the time (and later we can decide what to do about the errors).